lib/auth/oidc_test.go

```diff
@@ -840,7 +841,7 @@ func TestUsernameClaim(t *testing.T) {
 				RedirectURLs:  []string{"https://proxy.example.com/v1/webapi/oidc/callback"},
 				UsernameClaim: "prefred_usrnam",
 			},
-			expectedError: "The configured username_claim of 'prefred_usrnam' was not received from the IdP. Please update the username_claim in connector 'okta-oidc'.",
+			expectedError: "The configured username_claim of \"prefred_usrnam\" was not received from the IdP. Please update the username_claim in connector \"okta-oidc\".",
```

Comment:
This is perfect valid, but just for your awareness another option if you need `"` inside a string is to use backticks:

```
expectedError: `This is a string with "quotes" inside it`,
```